joining.html

```diff
@@ -0,0 +1,217 @@
+      <b>Orvolle I - Federation Navy Assembly Plant</b>. You will need to have
+      Teamspeak installed. The fleet commander will give you access to our
+      Teamspeak for the duration of the fleet.
+    </p>
+    <p>
+      You do not need to bring a ship, but if you want to, you are welcome to
+      bring anything cruiser-sized or smaller. If you do not have or want to
+      bring a ship, free ships appropriate for all SP levels are available.
+    </p>
+    <p>
+      The trial fleet generally lasts about <b>90 minutes</b>, and will drop you
+      off in Orvolle when it finishes.
+    </p>
+
+    <h2>
+      <a name="esi-knife" href="#esi-knife">#</a>
+      ESI Knife Audit
+    </h2>
+    <p>
+      Your recruiter will ask you to sign up for
+      <a href="https://esiknife.com/">ESI Knife</a>, which is a website that
+      allows your recruiter to look at your character's history, assets, and
+      so on. This is a standard security precaution for Eve corps and does
+      <b><u>not</u></b> grant any access to do anything other than view
+      details.
+    </p>
+
+    <h2>
+      <a name="interview" href="#interview">#</a>
+      Interview
+    </h2>
+    <p>
+      After your trial fleet, your recruiter will work with you to schedule
+      an interview, either in writing or over Teamspeak (or maybe both at once).
+      This interview generally lasts half an hour at most. You'll probably be
+      asked about at least these things:
+      <ul>
+        <li>Your experiences in Eve</li>
+        <li>Your corp history</li>
+        <li>Your trial fleet experience</li>
+      </ul>
+    </p>
+
+    <h2>
+      <a name="invitation" href="#invitation">#</a>
+      Invitation
+    </h2>
+    <p>
+      Once your trial fleet, ESI Knife audit, and interview are complete, your
+      recruiter will discuss your application with the other recruitment
+      staff. If everything looks good, your recruiter will invite you to join
+      the corp.
+    </p>
+
+    <h2>
+      <a name="joining" href="#joining">#</a>
+      Joining In Game
+    </h2>
+    <p>
+      If you're invited to join, you can now apply to join <b>Nobody in
+      Local</b> in game. A director will give your application a final check,
+      then accept your in-game application. Feel free to wait to confirm joining
+      in game if you need time to move assets or similar.
+    </p>
+
+    <h2>
+      <a name="onboarding" href="#onboarding">#</a>
+      Onboarding
+    </h2>
+    <p>
+      Once you've joined in game, head on over to <a href="onboarding.html">
+      onboarding</a> to get set up!
+    </p>
+
+    <h2>
+      <a name="questions" href="#questions">#</a>
+      Questions You Might Have
+    </h2>
+    <h3>
+      <a name="why-sp-max" href="#why-sp-max">#</a>
+      Why is there an SP maximum?
+    </h3>
+    <p>
+      Short answer: because we want SOUND to be a positive, friendly place for
+      low-SP and new players.
```

Comment:
```suggestion
      Short answer: because we want SOUND to be a positive, friendly place.
```

Maybe a bit too broad?

Reply:
yeah, I think that is a bit too broad - it doesn't really reference SP at all, which makes it a weird short answer to a question about an SP max :)

Reply:
But somehow it reflects on both our policies for <15M sp and >15M sp. Otherwise we need an answer for "why do I need a vouch when I have >15M sp?"

Reply:
Hm, that's a good point. Okay, accepted :)